bins/tts/train.py

```diff
@@ -20,6 +23,8 @@ def build_trainer(args, cfg):
         "VITS": VITSTrainer,
         "VALLE": VALLETrainer,
         "NaturalSpeech2": NS2Trainer,
+        "valle_v2_ar": valle_v2_ar,
```

Comment:
The naming format among "VITS", "VALLE", "NaturalSpeech2", "valle_v2_ar", ... should be consistent.

Reply:
renamed